auguste/roster.json

```diff
@@ -91,9 +91,14 @@
         }
       ]
     },
-    {
+      {
```

Comment:
Can you please move your opening curly brace for the rest of your code to be nicely indented?